src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -2216,6 +2216,10 @@ public void WritePartialHelloWorld(bool formatted, bool skipValidation)
         [InlineData(true, false, "<write base64 string when escape length bigger than given string")]
         [InlineData(false, true, "<write base64 string when escape length bigger than given string")]
         [InlineData(false, false, "<write base64 string when escape length bigger than given string")]
+        [InlineData(true, true, ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>)")]
+        [InlineData(true, false, ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")]
+        [InlineData(false, true, ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")]
+        [InlineData(false, false, ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")]
```

Comment:
Though I am fine with adding this case (especially since other test use it), was this test case necessary? I thought the case above that @buyaa-n added here https://github.com/dotnet/corefx/pull/39407 covered the "large property name that contains an escape character" case (specifically `"<write base64 string when escape length bigger than given string"`).

Reply:
Yeah, it should be same as `<write base64 string when escape length bigger than given string` by coverage around increasing buffer to accommodate escaped chars logic

Reply:
I added this case in order to complete coverage of 
`private void WriteBase64Indented(ReadOnlySpan<byte> escapedPropertyName, ReadOnlySpan<byte> bytes)` and
`private void WriteBase64Minimized(ReadOnlySpan<byte> escapedPropertyName, ReadOnlySpan<byte> bytes)`

![image](https://user-images.githubusercontent.com/1119735/61821349-3f5da400-ae57-11e9-8abe-9f74a5101729.png)
 and 
![image](https://user-images.githubusercontent.com/1119735/61821458-91062e80-ae57-11e9-93ac-6d71b2f0da78.png)

Other cases won't enter the if statement above.

Reply:
I see, great thank you @madmir